tests/app/main/views/test_broadcast.py

```diff
@@ -226,25 +226,26 @@ def test_choose_broadcast_library_page(
         service_id=SERVICE_ONE_ID,
         broadcast_message_id=fake_uuid,
     )
-    assert [
-        (normalize_spaces(title.text), normalize_spaces(hint.text))
-        for title, hint in list(zip(
-            page.select('.file-list-filename-large'), page.select('.file-list-hint-large')
-        ))
-    ] == [
-        (
+
+    titles = [
+        normalize_spaces(title.text)
+        for title in page.select('.file-list-filename-large')
+    ]
+    assert sorted(titles) == sorted([
             'Counties and Unitary Authorities in England and Wales',
-            'Barking and Dagenham, Barnet, Barnsley and 171 more…',
-        ),
-        (
             'Countries',
-            'England, Northern Ireland, Scotland and Wales',
-        ),
-        (
+            'Electoral Wards of the United Kingdom',
             'Regions of England',
-            'East Midlands, East of England, London and 6 more…',
-        ),
-    ]
+    ])
+
+    assert any([
+        all([
+            country in hint.text
+            for country in ('England', 'Northern Ireland', 'Scotland', 'Wales')
+        ])
+        for hint in page.select('.file-list-hint-large')
+    ])
```

Comment:
I’d prefer this test to be a little less clever as well:
```suggestion
    assert normalize_spaces(page.select('.file-list-hint-large')[1].text) == (
        'England, Northern Ireland, Scotland, and Wales'
    )
```